lib/report-portal-client.js

```diff
@@ -199,6 +200,38 @@ class RPClient {
         };
     }
 
+    /*
+     * @Returns {Object} - an object which contains a data for merge launches in ReportPortal.
+     */
+    getMergeLaunchesData(launchIds) {
+        return {
+            "description": "Merged launch",
+            "end_time": this.helpers.now(),
+            "extendSuitesDescription": true,
+            "launches": launchIds,
+            "merge_type": "BASIC",
+            "mode": "DEFAULT",
+            "name": this.config.launch || 'Test launch name',
+            "tags": this.config.tags
+        };
+    }
+
+    /*
+     * This method is used for merge launches in ReportPortal.
+     */
+    mergeLaunches() {
+        let url = [this.baseURL, "launch", "merge"].join("/");
```

Comment:
let => const

Reply:
I don't see any purpose to make it const. It is just variable, not constant.

Reply:
Linter did it